src/Data/Recruit.ChangeScripts/020-migrate-vacancyReviews_to_reviews.js

```diff
@@ -0,0 +1,29 @@
+{
+    //Migrating documents from 'vacancyReviews' to 'reviews'
+    //This script is idempotent
+    print("Migrating documents from 'vacancyReviews' to 'reviews'");
+
+    load("helper.js");
+
+    let matchedDocs = db.vacancyReviews.find();
+
+    while (matchedDocs.hasNext()) {
+        let doc = matchedDocs.next();
+        
+        if(db.reviews.findOne({"_id" : doc._id}) == null){
```

Comment:
You might need `===`

Could you print the count of source and target collections maybe.

Reply:
👍  Just spotted the the linting issue. It didn't like the reference to the external helper file.  I'll just remove it.  I'll also include your comments.